frontend/src/employee-frontend/components/fee-decisions/FeeDecisionsPage.tsx

```diff
@@ -49,16 +49,24 @@ const FeeDecisionsPage = React.memo(function FeeDecisionsPage() {
   const reloadDecisions = useRestApi(getFeeDecisions, setDecisionsResult)
 
   const {
-    feeDecisions: {
-      searchFilters,
-      debouncedSearchTerms,
-      checked,
-      toggleChecked,
-      checkIds,
-      clearChecked
-    }
+    feeDecisions: { searchFilters, debouncedSearchTerms }
   } = useContext(InvoicingUiContext)
 
+  const [checked, setChecked] = useState<Checked>({})
+  const toggleChecked = (id: string) =>
+    setChecked({
+      ...checked,
+      [id]: !checked[id]
+    })
+  const checkIds = (ids: string[]) => {
+    const idsChecked = ids.map((id) => ({ [id]: true }))
+    setChecked({
+      ...checked,
+      ...Object.assign({}, ...idsChecked)
+    })
+  }
+  const clearChecked = () => setChecked({})
+
```

Comment:
Since this piece of state is identical to the one used in `VoucherValueDecisionsPage` you could extract this to a custom hook.

Reply:
I put this in a separate file as it did not seem to fit with the Invoicing context stuff